sources/packages/backend/apps/queue-consumers/src/services/workflow/workflow-enqueuer.service.ts

```diff
@@ -58,6 +68,39 @@ export class WorkflowEnqueuerService {
     }
   }
 
+  /**
+   * Search applications with assessments to be cancelled.
+   * Enqueues the oldest assessment for the application with cancellation request status.
+   * @param summary process summary to group all the logs.
+   */
+  async enqueueCancelAssessmentWorkflows(
+    summary: ProcessSummary,
+  ): Promise<void> {
+    try {
+      summary.info(
+        "Checking database for applications with assessments waiting to be cancelled.",
+      );
+      const applications =
+        await this.applicationService.getApplicationsToCancelAssessments();
+      summary.info(`Found ${applications.length} applications.`);
+      if (!applications.length) {
+        return;
+      }
+      const children = await processInParallel(
```

Comment:
Should we also consider specifying the 3rd parameter: `maxParallelRequests` here?

Reply:
Hope that it was clarified but when not provided it will be set to the amount of CPUs.